src/components/include/policy/policy_external/policy/policy_manager.h

```diff
@@ -533,6 +533,12 @@ class PolicyManager : public usage_statistics::StatisticsManager,
    */
   virtual void OnAppsSearchCompleted(const bool trigger_ptu) = 0;
 
+  /**
+   * @brief Change applicatios count ready for PTU
```

Comment:
@ydementieiev typo

Reply:
@AKalinich-Luxoft done in [fe7b393](https://github.com/smartdevicelink/sdl_core/pull/3208/commits/fe7b393fe80f89ac9137d6dd9b41a0dfd6ba28c6)